app/assets/javascripts/host_edit.js

```diff
@@ -170,10 +170,10 @@ function submit_with_all_params() {
     type: 'POST',
     url: $('form').attr('action'),
     data: serializeForm(),
-    success: function(response) {
+    success: function(response, _responseStatus, _jqXHR) {
       // workaround for redirecting to the new host details page
       if (!response.includes('id="main"')) {
-        return tfm.nav.pushUrl('/new/hosts/' + fqdn);
+        return tfm.nav.pushUrl('/new/hosts/' + host_unique_name);
```

Comment:
do we need to wrap it with `foremanUrl` ?

Reply:
no, it is not used for anything useful, but it might in the future :)